compass_sdk/__init__.py

```diff
@@ -379,6 +393,7 @@ class ParseableDocument(BaseModel):
     content_length_bytes: PositiveInt  # File size must be a non-negative integer
     bytes: str  # Base64-encoded file contents
     context: Dict[str, Any] = Field(default_factory=dict)
+    heavy_parsing_strategy: HeavyParsingStrategy
```

Comment:
Provide a default if u want to make it backward compatible. Otherwise it will be a major version bump

Reply:
True, thanks